frontend/components/dm-list/dm_list.jsx

```diff
@@ -3,24 +3,23 @@ import DmListItem from './dm_list_item_container';
 
 class DmList extends React.Component {
 
-  render() {
-    const dms = Object.values(this.props.channels).filter(
+  createDmEls(convos){
+    const dms = Object.values(convos).filter(
```

Comment:
this method, while its cleaner than it was before, it can be cleaner. both iterations fit nicely in a single line.